cico_run_tests.sh

```diff
@@ -14,7 +14,7 @@ set -e
 # that might interest this worker.
 if [ -e "jenkins-env" ]; then
   cat jenkins-env \
-    | grep -E "(JENKINS_URL|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+    | grep -E "^(JENKINS_URL|REFRESH_TOKEN|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
```

Comment:
why do we need `^` at start there? build fails with syntax error. `/tmp/jenkins-env: line 11: syntax error near unexpected token `('`

Reply:
and please append new keys, easier to review :)

Reply:
>why do we need ^ at start there? build fails with syntax error. /tmp/jenkins-env: line 11: syntax error near unexpected token('`

Please check - https://chat.openshift.io/developers/pl/8cq4htg5y7fazcfogs46cpzf7a
and https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/1679